i18n/en.po

```diff
@@ -0,0 +1,660 @@
+msgid "CHANGE_PHONENUMBER_BUTTON"
+msgstr "Change your phone number"
+
+#: client/components/mobile-phone-change/mobile-phone-change.js:187
+msgid "CANCEL_BUTTON"
+msgstr "Cancel"
+
+#: client/components/logout/logout.js:23
+msgid "LOGOUT_PAGE_TITLE"
+msgstr "Logout"
+
+#: client/components/logout/logout.js:45
+msgid "LOGOUT_CONTENT"
+msgstr "Logout successful"
+
+#: client/components/logout/logout.js:52
+msgid "LOGIN_AGAIN"
+msgstr "Log in again"
+
+#: client/components/login/login.js:114
+msgid "USERNAME_TITLE"
+msgstr "enter valid email address"
+
+#: client/components/login/login.js:52
+msgid "LOGIN_PAGE_TITLE"
+msgstr "Log in"
+
+#: client/components/login/login.js:273
+#: client/components/login/login.js:277
+#, javascript-format
+msgid "SOCIAL_LOGIN_TEXT ${ link.text }"
+msgstr "Login with ${ link.text }"
+
+#: client/components/login/login.js:329
+msgid "REMEMBER_ME_LABEL"
+msgstr "remember me on this device"
+
+#: client/components/login/login.js:336
+msgid "LOGIN_ADDITIONAL_INFO_TEXT"
+msgstr ""
+"By logging in, you accept the {terms_and_conditions} and the "
+"{privacy_policy} of this WiFi service. "
+
+#: client/components/login/login.js:347
+msgid "LOGIN_TEXT"
+msgstr "Login"
+
+#: client/components/login/login.js:353
+msgid "REGISTER_BUTTON_LABEL"
+msgstr "Not registered yet?"
+
+#: client/components/login/login.js:355
+msgid "REGISTER_BUTTON_TEXT"
+msgstr "Sign up"
+
+#: client/components/header/header.js:114
+#: client/components/header/header.js:127
+#: client/components/header/header.js:182
+#: client/components/header/header.js:195
+#, javascript-format
+msgid "HEADER_LINK ${ link.text }"
+msgstr "${ link.text }"
+
+#: client/components/footer/footer.js:28
+#, javascript-format
+msgid "FOOTER_LINK ${ link.text }"
+msgstr "${ link.text }"
+
+#: client/components/footer/footer.js:37
+msgid "FOOTER_SECONDARY_TEXT"
+msgstr " "
+
+#: client/components/contact-box/contact.js:15
+#, javascript-format
+msgid "CONTACT_SOCIAL_LINK ${ link.alt }"
+msgstr "${ link.alt }"
+
+#: client/components/contact-box/contact.js:20
+msgid "CONTACT_EMAIL_LABEL"
+msgstr "E-mail"
+
+#: client/components/contact-box/contact.js:22
+msgid "CONTACT_EMAIL"
+msgstr "support@openwisp.co"
+
+#: client/components/contact-box/contact.js:27
+msgid "CONTACT_HELPDESK_LABEL"
+msgstr "Helpdesk"
+
+#: client/components/contact-box/contact.js:29
+msgid "CONTACT_HELPDESK"
+msgstr "+789 948 564"
+
+#: client/components/404/404.js:13
+msgid "404_PAGE_TITLE"
+msgstr "404 Not found"
+
+#: client/components/404/404.js:23
+msgid "404_PAGE_HEADING"
+msgstr "404"
+
+#: client/components/404/404.js:27
+msgid "404_PAGE_SUBHEADING"
+msgstr "Not Found"
+
+#: client/components/404/404.js:32
+msgid "404_PAGE_MESSAGE"
+msgstr "Sorry, we couldn't find the page you were looking for."
+
+#: client/components/404/404.js:38
+msgid "HOME_PAGE_LINK_TEXT"
+msgstr "Go back to homepage"
+
+#: client/components/status/status.js:602
+msgid "DATE_LANGUAGE_LOCALE"
+msgstr "en-US"
```

Comment:
how did this end up here? Does not look like a translatable string, it's just the language code, can you point out at what part of the code is generating this please?

Reply:
In status page for account start and stop time. Basically it translates the time to be in the format of that locale